exposed-jodatime/src/main/kotlin/org/jetbrains/exposed/sql/jodatime/DateFunctions.kt

```diff
@@ -32,17 +29,20 @@ class Time<T : DateTime?>(val expr: Expression<T>) : Function<LocalTime>(LocalTi
 }
 
 /**
- * Represents an SQL function that returns the current date and time, as [DateTime]
+ * Represents an SQL function that returns the current date and time, as [DateTime] with the specified fractional seconds [precision].
  *
  * @sample org.jetbrains.exposed.JodaTimeDefaultsTest.testDefaultExpressions02
  */
-object CurrentDateTime : Function<DateTime>(DateColumnType(true)) {
+open class CurrentDateTime(private val precision: Byte? = null) : Function<DateTime>(DateColumnType(true, precision)) {
     override fun toQueryBuilder(queryBuilder: QueryBuilder) = queryBuilder {
         +when {
-            (currentDialect as? MysqlDialect)?.isFractionDateTimeSupported() == true -> "CURRENT_TIMESTAMP(6)"
+            (currentDialect as? MysqlDialect)?.isFractionDateTimeSupported() == true ||
+                (currentDialect !is SQLiteDialect && currentDialect !is SQLServerDialect) -> "CURRENT_TIMESTAMP${precision?.let { "($it)" }.orEmpty()}"
             else -> "CURRENT_TIMESTAMP"
         }
     }
+
+    companion object : CurrentDateTime()
```

Comment:
After changing `CurrentDateTime` from an object to a class, this line enables users to still use it like this `CurrentDateTime` and not be forced to use it like this `CurrentDateTime()`, thereby avoiding a breaking change.